packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_account_utxos_to_account.test.ts

```diff
@@ -33,13 +33,7 @@ beforeEach(async () => {
   await container.waitForWalletBalanceGTE(101)
 
   // fund utxos balance
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 1.1) // 1.1
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 5.5) // 6.6
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 10.566) // 17.166
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 15.51345) // 32.67945
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 20) // 52.67945
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 37.98) // 90.65945
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 9.34055) // 100
```

Comment:
Given this test was previously written to make sure "utxo can join" successfully. Should we keep this in?

Reply:
ok added 
```
// fund utxos balance
  await fundEllipticPair(container, providers.elliptic.ellipticPair, 10) // 10
  await fundEllipticPair(container, providers.elliptic.ellipticPair, 10) // 20
```
to check utxo aggregation when required. Had to do this way since deterministic values are required  to write end `expects`